.github/workflows/test.yml

```
@@ -26,15 +26,24 @@ jobs:
       fail-fast: false
       matrix:
         config:
+          # On Windows,
+          #
+          # * for R >= 4.2, both the MSVC toolchain and the GNU toolchain should
+          #   work. Since our main support is on MSVC, we mainly test MSVC here.
+          #   Also, at least one GNU case should be included so that we can
+          #   detect when something gets broken. 
+          # * for R < 4.2, the MSVC toolchain must be used to support
+          #   cross-compilation for the 32-bit. 
+          # 
           # options:
           #   - targets: Targets to build and run tests against. If not specified, 'default' will be used. 
           #   - no-test-targets: Targets to skip tests.
-          - {os: windows-latest, r: 'release', rust-version: 'stable-gnu',  targets: ['x86_64-pc-windows-gnu'], rtools-version: '42'}
-          - {os: windows-latest, r: 'release', rust-version: 'nightly-gnu', targets: ['x86_64-pc-windows-gnu'], rtools-version: '42'}
-          - {os: windows-latest, r: 'devel',   rust-version: 'stable-gnu',  targets: ['x86_64-pc-windows-gnu'], rtools-version: '42'}
-          # For R < 4.2, the MSVC toolchain is used to support cross-compilation for the 32-bit.
-          # Change this to the GNU toolchain as well when 'oldrel' points to R 4.2 or above.
-          - {os: windows-latest, r: 'oldrel',  rust-version: 'stable-msvc', targets: ['x86_64-pc-windows-gnu', 'i686-pc-windows-gnu']}
+          #   - emit-bindings: do not emit bindings if 'false'
```

Let's turn this statement around, it should be easier to read
```suggestion
          #   - emit-bindings: if 'true', emit bindings
```

---

`emit-bindings` emits bindings even when it's not explicitly specified `'true'`, so I think this needs to be improved. Anyway, I'll do it in another pull request so I'm fine with your suggestion.